bom/pom.xml

```diff
@@ -235,11 +235,13 @@
         <version>4.12</version>
       </dependency>
       <!-- AssertJ for smart assertions -->
+      <!-- 
```

Comment:
If a dependency is not required please delete it, don't keep it polluting the pom.xml

unless is used as a template or example for some use, and in that case it should be properly configured.